src/GeneralStateTestsFiller/stRandom/randomStatetest646Filler.json

```diff
@@ -0,0 +1,53 @@
+{
+    "randomStatetest646" : {
+        "_info" : {
+            "comment" : "Geth Failed this test on all networks"
```

Comment:
ok this test is fixed. the issue was in cpp applying zeroAddress rules on all forks.